lib/presenter/Chart.js

```diff
@@ -98,8 +98,9 @@ class Chart extends Events {
         this._aggregateDataRange.on('change', (e) => this.throttledUpdate());
 
         if (this.resizeWidth || this.resizeHeight) {
+            this.resizeEventID = Utils.createUniqueId('resize.chart');
             this.throttledResize = this.throttledResize || Utils.throttle(this.onWindowResize, 100);
-            d3.select(window).on('resize.chart', () => {
+            d3.select(window).on(this.resizeEventID, () => {
```

Comment:
Each chart binds to its own `resizeEventID` so that multiple resizable charts can exist on the same page